### DISCUSSION TITLE
Repo Deployment

### DISCUSSION THREAD

**Discussion by Emmynando** [community/community#35977] [Other Feature Feedback, Questions, & Ideas]
i keep having a time-out session each time i intend to deploy my repo on netlify

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Emmynando ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
